docs/content/troubleshooting/troubleshooting-with-app-protect.md

```diff
@@ -73,7 +73,7 @@ Note that in the events section, we have a `Normal` event with the `AddedOrUpdat
 
 ### Replace the policy
 
-NOTE: This method only applies if using [external references](https://docs.nginx.com/nginx-app-protect/configuration/#external-references) 
+NOTE: This method only applies if using [external references](/nginx-app-protect/configuration/#external-references) 
```

Comment:
Wasn't able to test this link in the preview; got a 404.

Reply:
same as above

Reply:
same as above